source/components/defining-a-component.md

```diff
@@ -55,9 +81,9 @@ changes to the component's element using JavaScript.
 
 Ember knows which subclass powers a component based on its name. For
 example, if you have a component called `blog-post`, you would create a
-subclass called `App.BlogPostComponent`. If your component was called
-`audio-player-controls`, the class name would be
-`App.AudioPlayerControlsComponent`.
+a file at `app/components/blog-post.js`. If your component was called
```

Comment:
It looks from the diff that this will end up reading "you would create a a file", but I could be mis-reading the diff.  Can you double check?